filers/default/src/main/java/org/commonjava/indy/filer/def/DefaultGalleyStorageProvider.java

```diff
@@ -198,7 +198,7 @@ private void setupCacheProviderFactory()
             // Only work for local debug mode.
             ScheduledExecutorService debugDeleteExecutor = Executors.newScheduledThreadPool( 5, new NamedThreadFactory(
                     "debug-galley-delete-executor", new ThreadGroup( "debug-galley-delete-executor" ), true, 2 ) );
-            cacheProviderFactory = new PartyLineCacheProviderFactory( storeRoot, debugDeleteExecutor );
+            cacheProviderFactory = new PartyLineCacheProviderFactory( storeRoot, timeoutProcessing, debugDeleteExecutor );
```

Comment:
where is this timeoutProcessing declared?

Reply:
@ligangty yeah, typo, updated